_posts/en/newsletters/2025-05-16-newsletter.md

```diff
@@ -0,0 +1,150 @@
+Bitcoin's consensus rules._
+
+- **Proposed BIP for 64-bit arithmetic in Script:** Chris Stewart
+  [posted][stewart bippost] a [draft BIP][64bit bip] to the Bitcoin-Dev
+  mailing list that proposes upgrading Bitcoin's existing opcodes to
+  operate on 64 numeric values.  This follows his previous research into
+  the subject (see Newsletters [#285][news285 64bit], [#290][news290
+  64bit], and [#306][news306 64bit]).  In a change from some of the
+  previous discussion, the new proposal uses numbers in the same
+  compactSize data format currently used in Bitcoin.  Additional related
+  [discussion][stewart inout] occurred on two [threads][stewart
+  overflow] on Delving Bitcoin.
+
+<!-- FIXME:harding: ran out of time, will work on some of these topics
+
+- Bram Cohen quines
+- CTV+CSFS BitVM bridges
+- General computation with OP_CAT
+- Wager on OP_CAT activation
+- Post quantum signatures: https://mailing-list.bitcoindevs.xyz/bitcoindev/76c5ec26-5fd5-4746-86ed-89d2c8e28cbc@mattcorallo.com/T/#t
+- Substring search: https://mailing-list.bitcoindevs.xyz/bitcoindev/e1c28747-ccc9-4d14-a93f-77dbe08a0ba9n@googlegroups.com/T/#t
+- Path to general computation on Bitcoin with OP_CAT https://delvingbitcoin.org/t/the-path-to-general-computation-on-bitcoin-with-op-cat/1106
+- Post-sig cross-input scripting: https://delvingbitcoin.org/t/post-signature-cross-input-scripting-using-the-taproot-annex/1520/3
+
+-->
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+- [LND 0.19.0-beta.rc4][] is a release candidate for this popular LN
+  node.  One of the major improvements that could probably use testing
+  is the new RBF-based fee bumping for cooperative closes.
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #32155][] updates the internal miner to [timelock][topic
+  timelocks] coinbase transactions by setting the `nLockTime` field to the
+  current block’s height minus one and requiring the `nSequence` field not to be
+  final (to enforce the timelock). Although the built-in miner isn’t typically
+  used on mainnet, updating it encourages mining pools to adopt these changes
+  early in their own software in preparation for the proposed [BIP54][]
+  [consensus cleanup]​[topic consensus cleanup] soft fork. Timelocking coinbase
```

Comment:
There is a zero-width space between `][` which prevents link from forming. Suggestion does not work, must be cleaned up manually.

Reply:
Deleted the space and rephrased slightly per our policy of trying to avoid two links in a row (which visually appear to be a single link).  Thanks!